gd1_helpers/membership/gd1_model.py

```diff
@@ -8,67 +8,81 @@
     Normal1DSplineMixtureVariable,
     Normal1DSplineVariable,
     UniformVariable,
+    Normal1DVariable
 )
 
-phi1_lim = (-100, 20)
+#phi1_lim = (-100, 20)
```

Comment:
Why the change to (-20, 20) in the base class below?

Reply:
I think I needed to have a `phi1_lim` during the import of the Base class and I wanted to choose something that would be a default for most streams. It automatically gets updated with the extent of the data once you run `setup` though